proto/minder/v1/minder.proto

```diff
@@ -2082,7 +2084,9 @@ message ListRuleTypesRequest {
 // ListRuleTypesResponse is the response to list rule types.
 message ListRuleTypesResponse {
     // rule_types is the list of rule types.
-    repeated RuleType rule_types = 1;
+    repeated RuleType rule_types = 1 [
+        (google.api.field_behavior) = REQUIRED
+    ];
```

Comment:
will this work even if we get an empty set of rule types?

Reply:
Yes empty set will work, but nil won't work. I tested the current implementation and it's already defaulting to an empty set, even though it's not explicitly set in the code.